docs/development.md

```diff
@@ -14,6 +14,12 @@
 
 ## Setting up development environment
 
+- Prerequisite for development environment
+
+```
+yarn install
```

Comment:
could add this here:

Install Postgres on your local machine. If you would like to use a different database, you may need to update the `config/database.yml` file.